webapp/src/views/projects/translations/Namespace/NamespaceContent.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+import { Menu, MenuItem, styled } from '@mui/material';
+import { ArrowDropDown } from '@mui/icons-material';
+import { useTranslate } from '@tolgee/react';
+
+import { useNamespaceFilter } from './useNamespaceFilter';
+import { NamespaceRenameDialog } from './NamespaceRenameDialog';
+import { NsBannerRecord } from '../context/useNsBanners';
+import { useProjectPermissions } from 'tg.hooks/useProjectPermissions';
+import { ProjectPermissionType } from 'tg.service/response.types';
+
+const StyledNamespace = styled('div')`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  background: ${({ theme }) => theme.palette.background.default};
+  padding: ${({ theme }) => theme.spacing(0, 1.5, 0, 1.5)};
+  padding-bottom: 1px;
+  height: 24px;
+  position: relative;
+  top: -4px;
+  border-radius: 12px;
+  box-shadow: ${({ theme }) =>
+    theme.palette.mode === 'dark'
+      ? '0px 0px 7px -1px #000000'
+      : '0px 0px 7px -2px #000000'};
+  z-index: 1;
+  max-width: 100%;
+`;
+
+const StyledContent = styled('div')`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const StyledMoreArrow = styled('div')`
+  display: flex;
+  align-items: center;
+  padding-left: 2px;
+  margin-right: ${({ theme }) => theme.spacing(-0.5)};
+`;
+
+type Props = {
+  namespace: NsBannerRecord;
+  sticky?: boolean;
+};
+
+export const NamespaceContent = React.forwardRef<HTMLDivElement, Props>(
+  function NamespaceContent({ namespace, sticky }, ref) {
+    const t = useTranslate();
+    const { toggle, isActive } = useNamespaceFilter(namespace.name);
+    const [open, setOpen] = useState<undefined | HTMLElement>(undefined);
+    const [renameOpen, setRenameOpen] = useState(false);
+    const permission = useProjectPermissions();
+    const canRename = permission.satisfiesPermission(
+      ProjectPermissionType.EDIT
+    );
+
+    const handleClose = () => {
+      setOpen(undefined);
+    };
+
+    return (
+      <>
+        <StyledNamespace
+          ref={ref}
+          onClick={toggle}
```

Comment:
The onclick shouldn't filter it, but open the dropdown. User can choose it there. There should be some hover as there is hover for tags.